h/static/scripts/annotation-ui.js

```diff
@@ -86,18 +136,12 @@ module.exports = function (settings) {
           selection[annotations[i].id] = true;
         }
       }
-      this.selectedAnnotationMap = value(selection);
+      select(selection);
     },
 
-    /**
-     * @ngdoc method
-     * @name annotationUI.xorSelectedAnnotations()
-     * @returns nothing
-     * @description takes an array of annotations and adds them to the
-     * selectedAnnotationMap if not present otherwise removes them.
-     */
+    /** Toggle whether annotations are selected or not. */
```

Comment:
I think just one more line in this comment explaining that for each annotation in `annotations`, its selected state will be toggled, would be helpful.

In addition, although I appreciate you're trying to keep the size of the PRs down, `xorSelectedAnnotations` is a rather unfortunate name. It looks like replacing `xor` with `toggle` would be relatively straightforward and would IMO be a great improvement.